login/service_whitebox_test.go

```diff
@@ -204,16 +202,14 @@ func TestInt64ToInt64OK(t *testing.T) {
 }
 
 func TestFloat32ToInt64OK(t *testing.T) {
-	var f32 float32
-	f32 = 0.1e1
+	f32 := 0.1e1
 	i, err := numberToInt(f32)
 	assert.Nil(t, err)
 	assert.Equal(t, int64(f32), i)
 }
 
 func TestFloat64ToInt64OK(t *testing.T) {
-	var f64 float64
-	f64 = 0.1e10
+	f64 := 0.1e10
```

Comment:
thanks,
have done in https://github.com/fabric8-services/fabric8-wit/pull/1489/commits/11a85853665905840341dc5a1df3545d84c0eaea#diff-ce0aa7f266a3f01ad173ef2e98624125R215